src/components/DivElement.js

```diff
@@ -1,6 +1,13 @@
 const HTMLElement = require('./HTMLElement');
-
 // Define class here
-
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super('div', content);
+    super.render();
+  }
+}
+const andIThinkToMyself = new DivElement('What a wonderful world');
+console.log(andIThinkToMyself.render());
+// prints "<div>What a wonderful world</div>"
 // Export class here
 module.exports = {};
```

Comment:
Remember to export this class at the end of the file. At the moment this file is exporting an empty object.